tests/test_views.py

```diff
@@ -30,15 +30,17 @@ def setUp(self):
         self.record1 = Record.objects.create(occurred_from=self.now,
                                              occurred_to=self.now,
                                              geom='POINT (0 0)',
+                                             nominatim='Equator',
                                              schema=self.schema)
         self.record2 = Record.objects.create(occurred_from=self.then,
                                              occurred_to=self.now,
                                              geom='POINT (0 0)',
+                                             nominatim='Equator',
                                              schema=self.schema)
 
     def test_toddow(self):
         url = '/api/records/toddow/?record_type={}'.format(str(self.record_type.uuid))
-        response_data = json.loads(self.client.get(url).content)[1]  # only one record to count
+        response_data = json.loads(self.client.get(url).content)[0]  # only one record to count
```

Comment:
This is a duplicate of something that has already been merged: https://github.com/azavea/ashlar/commit/51427cf57cb2a90317cbd575aa9e893f40146801 ; the best way to deal with that is probably to drop this commit and rebase from the latest develop before merging to confirm tests still work.